vendor: bump Pebble to 2215b8d4c8ab

jbowens opened the pull request:
```
2215b8d4 tool: include version edit index in dump output
9015edae Ensure EventListener functions set all callbacks
2c03ea59 make number of table cache shards configurable
0aa9fa9f tool: add --edit-count flag to LSM visualizer
271d6cd0 record: move package out of internal directory
ff73aaba Add Github actions
6f23da1c Clean up travis CI config
a0e92d97 compaction: clean up useless code
b7ed27e6 travis: add go 1.16 CI
```

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/67502)
<!-- Reviewable:end -->

andyyang890 reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> 

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/67502#-:-MeQrP-n32yBdB4jad2c:bg7igkv)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @itsbilal and @sumeerbhola)



<!-- Sent from Reviewable.io -->

jbowens committed:
vendor: bump Pebble to 2215b8d4c8ab

2215b8d4 tool: include version edit index in dump output
9015edae Ensure EventListener functions set all callbacks
2c03ea59 make number of table cache shards configurable
0aa9fa9f tool: add --edit-count flag to LSM visualizer
271d6cd0 record: move package out of internal directory
ff73aaba Add Github actions
6f23da1c Clean up travis CI config
a0e92d97 compaction: clean up useless code
b7ed27e6 travis: add go 1.16 CI

Release note: None

jbowens reviewed (commented):
TFTR!

bors r+

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/67502#-:-MeVm-kN-uiLo79ZLDwM:b-oduykh)* status: :shipit: complete! 0 of 0 LGTMs obtained (and 1 stale) (waiting on @itsbilal and @sumeerbhola)



<!-- Sent from Reviewable.io -->
